doc/reference/modules/nhibernate_mapping_attributes.xml

```diff
@@ -165,7 +165,7 @@ public class Base {
 				<para>Instead of using a string for <methodname>DiscriminatorValue</methodname> (in <classname>[Class]</classname> and <classname>[Subclass]</classname>), you can use any object you want. Example: <programlisting>[Subclass(DiscriminatorValueEnumFormat="d", DiscriminatorValueObject=DiscEnum.Val1)]</programlisting> Here, the object is an Enum, and you can set the format you want (the default value is "g"). Note that you must put it <emphasis role="strong">before</emphasis>! For others types, It simply use the <methodname>ToString()</methodname> method of the object.</para>
 			</listitem>
 			<listitem>
-				<para>If you are using members of the type <classname>Nullables.NullableXXX</classname> (from the library <link linkend="nullables">Nullables</link>), then they will be mapped to <classname>Nullables.NHibernate.NullableXXXType</classname> automatically; don't set <literal>Type="..."</literal> in <classname>[Property]</classname> (leave it null). This is also the case for <classname>SqlTypes</classname> (and you can add your own patterns). Thanks to <emphasis>Michael Third</emphasis> for the idea :)</para>
+				<para>If you are using members of the type <classname>Nullables.NullableXXX</classname> (from the library Nullables), then they will be mapped to <classname>Nullables.NHibernate.NullableXXXType</classname> automatically; don't set <literal>Type="..."</literal> in <classname>[Property]</classname> (leave it null). This is also the case for <classname>SqlTypes</classname> (and you can add your own patterns). Thanks to <emphasis>Michael Third</emphasis> for the idea :)</para>
```

Comment:
I think this has to be removed completely. As this, probably does not happen currently. And, nobody is using Nullables library.